### PR TITLE
[terra-toggle] added Accessibility Guide 

### PR DESCRIPTION
**Pull request by udaymattam**
### Summary

Added Accessibility Guide For Terra-Toggle
Closes #

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by JessieRandle** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please include a CHANGELOG entry for each changed package this PR. Looks like a CHANGELOG is missing for: 

 - terra-core-docs</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against dae899d3b889aa978c90355e17e3f1f14599f17c
</p>

File: packages/terra-core-docs/src/terra-dev-site/doc/toggle/AccessibilityGuide.4.doc.mdx [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+<Notice variant="important" ariaLevel="3">
+
+### Code Considerations: What do engineers need to do?
+
+- Ensure all interactive elements within Terra Toggle have a meaningful label
+  - The Toggle button provides more information about a “thing”. Ensure the Toggle is programmatically associated with the “thing” it provides more information about.
+
+<Whitespace />
+
+<div>
```
Comment (supreethmr) [inline packages/terra-core-docs/src/terra-dev-site/doc/toggle/AccessibilityGuide.4.doc.mdx:26]: why do we need wrapping <div> here

File: packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/TogglePatient.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    if (event.nativeEvent.keyCode === KEY_SPACE || event.nativeEvent.keyCode === KEY_RETURN) {
+      setIsOpen(!isOpen);
+      event.preventDefault();
+    }
+  };
+
+  return (
+    <div>
+      <label htmlFor="Icon-label">
+        <IconInformation data-show-focus-styles={focused} focusable onKeyDown={handleOnKeyDown} onMouseDown={() => { setFocus(false); }} onClick={handleOnClick} a11yLabel="Patient, Information Icon" tabIndex="0" role="button" aria-expanded={isOpen} aria-controls="toggle" id="Icon-label" />
```
Comment (supreethmr) [inline packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/TogglePatient.jsx:27]:
```suggestion
        <IconInformation data-show-focus-styles={focused} focusable onKeyDown={handleOnKeyDown} onMouseDown={() => { setFocus(false); }} onClick={handleOnClick} a11yLabel="Allergies, Information Icon" tabIndex="0" role="button" aria-expanded={isOpen} aria-controls="toggle" id="Info-Icon" />
```

File: packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/TogglePatient.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    }
+    if (event.nativeEvent.keyCode === KEY_SPACE || event.nativeEvent.keyCode === KEY_RETURN) {
+      setIsOpen(!isOpen);
+      event.preventDefault();
+    }
+  };
+
+  return (
+    <div>
+      <label htmlFor="Icon-label">
```
Comment (supreethmr) [inline packages/terra-core-docs/src/terra-dev-site/doc/toggle/example/TogglePatient.jsx:26]:
```suggestion
      <label htmlFor="Info-Icon">
```

**Review by supreethmr — APPROVED** [review]

**Comment by JessieRandle** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please include a CHANGELOG entry for each changed package this PR. Looks like a CHANGELOG is missing for: 

 - terra-action-header
 - terra-alert
 - terra-button-group
 - terra-search-field
 - terra-status-view
 - terra-toggle-button</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 88c22919fe5f7e73782389a7ec955132a1f61866
</p>

**Pull request merged**